javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -105,6 +105,7 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
         new AgentBuilder.Default()
             .disableClassFormatChanges()
             .with(AgentBuilder.RedefinitionStrategy.RETRANSFORMATION)
+            .with(AgentBuilder.RedefinitionStrategy.DiscoveryStrategy.Reiterating.INSTANCE)
```

Comment:
Interesting... why did you add this?

Reply:
Oh yes, I had meant to point this out (nice find 😄), this fixed the guava addListener test that was left commented out in `CassandraClientTest`